plugins/sonar-xoo-plugin/src/main/java/org/sonar/xoo/XooPlugin.java

```diff
@@ -140,12 +139,12 @@ public void define(Context context) {
       XooProjectBuilder.class,
       XooPostJob.class);
 
-    if (context.getRuntimeProduct() != SonarProduct.SONARLINT) {
+    if (context.getRuntime().getProduct() != SonarProduct.SONARLINT) {
       context.addExtensions(MeasureSensor.class,
         DeprecatedResourceApiSensor.class);
     }
 
-    if (context.getSonarQubeVersion().isGreaterThanOrEqual(V5_5)) {
+    if (context.getRuntime().getApiVersion().isGreaterThanOrEqual(Version.create(5, 5))) {
```

Comment:
This is just to test the feature.
if (context.getRuntime().getApiVersion().isGreaterThanOrEqual(Version.create(5, 5)))
is totally useless since context.getRuntime() was introduced in 6.0 :)